src/main/java/com/kodilla/ecommercee/entity/Cart.java

```diff
@@ -1,16 +1,13 @@
 package com.kodilla.ecommercee.entity;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.*;
 
 import javax.persistence.*;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
 
 
+@EqualsAndHashCode
```

Comment:
Tak jak wspominaliśmy. Jesli ma byc equals i hascode na encji to powinien tylko używać ID